AuroraEditor/Application/AppDelegate.swift

```diff
@@ -38,6 +39,8 @@ final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
 
     private var updateModel: UpdateObservedModel = .shared
 
+    var cancellable = Set<AnyCancellable>()
```

Comment:
Is the cancellable called?

Reply:
When subscribing to `RecentProjectsStore.shared.$paths` to construct the recents menu when the paths change, I store the canellable in this set.